awx_collection/plugins/modules/tower_label.py

```diff
@@ -12,97 +13,92 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: tower_label
 author: "Wayne Witzel III (@wwitzel3)"
 version_added: "2.3"
-short_description: create, update, or destroy Ansible Tower label.
+short_description: create, update, or destroy Ansible Tower labels.
 description:
     - Create, update, or destroy Ansible Tower labels. See
       U(https://www.ansible.com/tower) for an overview.
 options:
     name:
       description:
-        - Name to use for the label.
+        - Name of this label.
+      required: True
+      type: str
+    new_name:
+      description:
+        - Setting this option will change the existing name (looked up via the name field).
       required: True
       type: str
     organization:
       description:
-        - Organization the label should be applied to.
+        - Organization this label belongs to.
       required: True
       type: str
-    state:
```

Comment:
Did the `state` parameter get fully removed?

That should be added to the list of backward-incompatible changes, preferably at the end of the list, since that should eliminate conflicts with https://github.com/ansible/awx/pull/6326

Reply:
I think the use that will affect the most are people passing `state: present`

Reply:
Backwards compatibility... you convinced me to put state back in. I had taken it out because it can only be "present" but I'll leave it in so it old playbooks using it won't break.